browser/ui/views/frame/vertical_tab_strip_container.h

```diff
@@ -0,0 +1,51 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_UI_VIEWS_FRAME_VERTICAL_TAB_STRIP_CONTAINER_H_
+#define BRAVE_BROWSER_UI_VIEWS_FRAME_VERTICAL_TAB_STRIP_CONTAINER_H_
+
+#include "chrome/browser/ui/views/frame/tab_strip_region_view.h"
+
+namespace views {
+class ScrollView;
+}
+
+// Wraps TabStripRegion and show it vertically.
+class VerticalTabStripContainer : public views::View {
```

Comment:
this class is still not named according to convention. We don't really care that it wraps TabStripRegionView (not TabStripRegion) so I think this should be VerticalTabStripRegionView

Reply:
Yup. That looks better. Thanks.